qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -18,3 +17,8 @@
             sandbox_error_message = "Setting CPU affinity failed: Operation not permitted"
         - sandbox_off:
             qemu_sandbox = off
+    variants operation:
+        - @default:
+            second_cpu-affinity = "1-3"
```

Comment:
It looks strange that no `first_cpu-affinity` and start with `second`.
I prefer to use the same style in the cfg, convert it to different style for different usage.
```
def convert_affinity(affinity):
    """
    convert the cpu affinitys between list (ex: [1, 2, 3]) and qemu-kvm command line style (ex: 1-3)
    """
    if isinstance(affinity, str):
        start, end = affinity.split('-')
        output = list(range(int(start), int(end) + 1))
    elif isinstance(affinity, list):
        if len(affinity) == 1:
            output = str(affinity[0])
        else:
            output = "%s-%s" % (affinity[0], affinity[-1])
    return output
```

Reply:
Updated, thanks!